lib/getAssetInfo.ts

```diff
@@ -0,0 +1,42 @@
+import prisma from "@/lib/prisma";
+import { env } from "process";
+
+export async function getAssetPath(id: string) {
+  const directusFileItem = await prisma.directusFiles.findFirst({
```

Comment:
Use the new naming convention:
```suggestion
  const directusFile = await prisma.directusFiles.findFirst({
```